articles/spring-apps/tutorial-managed-identities-key-vault.md

```diff
@@ -93,14 +93,13 @@ export SERVICE_IDENTITY=$(az spring app show --name "springapp" -s "myspringclou
 
 ### [User-assigned managed identity](#tab/user-assigned-managed-identity)
 
-First, create a user-assigned managed identity in advance with its resource ID set to `$USER_IDENTITY_RESOURCE_ID`.
+First, create a user-assigned managed identity in advance with its resource ID set to `$USER_IDENTITY_RESOURCE_ID`. Save the client ID for property configuration below.
 
-:::image type="content" source="media/tutorial-managed-identities-key-vault/app-user-managed-identity-key-vault.png" alt-text="Screenshot of Azure portal showing the Managed Identity Properties screen with 'Principle I D' and 'Client I D' highlighted." lightbox="media/tutorial-managed-identities-key-vault/app-user-managed-identity-key-vault.png":::
+:::image type="content" source="media/tutorial-managed-identities-key-vault/app-user-managed-identity-key-vault.png" alt-text="Screenshot of Azure portal showing the Managed Identity Properties screen with 'Resource I D', 'Principle I D' and 'Client I D' highlighted." lightbox="media/tutorial-managed-identities-key-vault/app-user-managed-identity-key-vault.png":::
```

Comment:
Can we use `ID` instead of `I D`? I mean delete the space between `I` and `D`.

Reply:
I also noticed this usage, let me remove it.

Reply:
I did that at one point for alt-text values so that screen readers will read the letters separately. However, the guidance now states that this is unnecessary for capitalized acronyms.

Reply:
@KarlErickson Thanks for sharing the information.

> However, the guidance now states that this is unnecessary for capitalized acronyms.

Could you please share the link to the `guidance`?

Reply:
@chenrujun [Accessibility guidelines for images and videos](https://review.docs.microsoft.com/en-us/help/contribute/contribute-accessibility-multimedia?branch=main)